settings.gradle

```diff
@@ -13,6 +13,8 @@ include ':debug'
 include ':trustchain-trader'
 include ':trustchain-payloadgenerator'
 include ':currencyii'
+include ':FOC'
```

Comment:
Probably an extra line here, FOC module has been renamed to freedomOfComputing

Reply:
fixed, thanks